src/payment/strategies/affirm/affirm-payment-strategy.ts

```diff
@@ -91,6 +90,13 @@ export default class AffirmPaymentStrategy implements PaymentStrategy {
                     paymentData: { nonce: result.checkout_token },
                 };
 
+                if (result.checkout_token === 'payment_canceled') {
+                    return this._store.dispatch(
```

Comment:
The indentation is wrong, as the `catch` applies to the `dispatch` call rather than `submitPayment`. It should be something like this:

```ts
return this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload))
    .catch(() => {
        throw new PaymentMethodCancelledError();
    });
```